Automattic/wp-calypso
# Jetpack Free in most popular plans experiment

### Changes proposed in this Pull Request

This PR implements the new pricing page experiment. It moves Jetpack Free to the most popular section, and CRM Free to the bottom of the page.

Fixes 1196341175636977-as-1199527862646261 

### Testing instructions

- Download the PR and run Calypso and cloud concurrently
- For both platforms, check the following
- Select the control version (`withoutFree_control`) of the experiment (`jetpackPopularPlans`)
- Check that the plans/pricing page looks and behaves like production
- Select the test version (`withFree_test`) of the experiment
- Check the order of the most popular section (Free, Security Daily, Complete)
- Check the order of the more products section (Backup Daily, Backup Realtime, Security Realtime, etc.)
- Check the content of the Jetpack Free card against the mockups (see linked task)
- Check that the CRM Free card takes the full width of the cards container
- Check the content of the CRM Free card against the mockups

### Screenshots

_Jetpack Free is part of the most popular section_
<img width="1072" alt="Screen Shot 2021-03-03 at 3 33 43 PM" src="https://user-images.githubusercontent.com/1620183/109871923-b6fdd280-7c39-11eb-8cc5-d2a91b6d71b6.png">

_Backpu Daily is moved to the next section_
<img width="1073" alt="Screen Shot 2021-03-03 at 3 33 50 PM" src="https://user-images.githubusercontent.com/1620183/109871987-ced55680-7c39-11eb-8a82-71b5554c81a9.png">

_CRM Free is moved to the bottom of the page_
<img width="1070" alt="Screen Shot 2021-03-03 at 3 34 01 PM" src="https://user-images.githubusercontent.com/1620183/109872034-e01e6300-7c39-11eb-9211-8067ac04e9fc.png">
